.editorconfig

```diff
@@ -60,14 +60,6 @@ dotnet_naming_rule.interface_names_must_begin_with_I.symbols   = interfaces
 dotnet_naming_rule.interface_names_must_begin_with_I.style = pascal_begin_with_I_style
 dotnet_naming_rule.interface_names_must_begin_with_I.severity = warning
 
-dotnet_naming_rule.private_and_protected_fields_must_begin_with_underscore.symbols   = private_fields
```

Comment:
The editorconfig was correct - it's the code that is behind. I'd like private members to be `_likeThis` and public members to be `LikeThis`.